examples/celery-rabbitmq/flower-controller.yaml

```diff
@@ -18,3 +18,10 @@ spec:
         resources:
           limits:
             cpu: 100m
+        livenessProbe:
+          httpGet:
+            # Path to probe; should be cheap, but representative of typical behavior
+            path: /
+            port: 80
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
```

Comment:
Isn't it first waiting 30secs and then queries the endpoint every second after that?
I think that would be fine.